grazer/src/main/java/edu/asu/diging/grazer/config/SecurityContext.java

```diff
@@ -24,25 +24,25 @@ public void configure(WebSecurity web) throws Exception {
     @Override
     protected void configure(HttpSecurity http) throws Exception {
        http.formLogin()
-                .loginPage("/")
-                .loginProcessingUrl("/login/authenticate")
-                .failureUrl("/?error=bad_credentials")
+                .loginPage("/login")
+                //.loginProcessingUrl("/login/authenticate")
+                .failureUrl("/loginfailed")
                 // Configures the logout function
                 .and()
                 .logout()
                 .deleteCookies("JSESSIONID")
                 .logoutUrl("/logout")
                 .logoutSuccessUrl("/")
-                .and().exceptionHandling().accessDeniedPage("/403")
+                .and().exceptionHandling().accessDeniedPage("/403");
                 // Configures url based authorization
-                .and()
-                .authorizeRequests()
+                //.and()
+                //.authorizeRequests()
                 // Anyone can access the urls
-                .antMatchers("/", "/persons/**", "/person/**", "/concept/**", "/query**", "/info", "/resources/**", "/login/authenticate", "/rest/**",
-                        "/logout").permitAll()
+                //.antMatchers("/", "/persons/**", "/person/**", "/concept/**", "/query**", "/info", "/resources/**", "/login/authenticate", "/rest/**",
+                //        "/logout").permitAll()
                 // The rest of the our application is protected.
-                .antMatchers("/users/**", "/admin/**", "/api/**").hasRole("ADMIN")
-                .anyRequest().hasRole("USER").and().httpBasic();
+                //.antMatchers("/users/**", "/admin/**", "/api/**").hasRole("ADMIN")
+                //.anyRequest().hasRole("USER").and().httpBasic();
```

Comment:
that seems wrong. we need some pages to be accessed by everyone, some only by admins. why did you comment that all out?